arches_references/src/arches_references/components/misc/ListHeader.vue

```diff
@@ -22,6 +24,7 @@ const { displayedRow } = inject(displayedRowKey) as unknown as {
     displayedRow: Ref<Selectable>;
 };
 const selectedLanguage = inject(selectedLanguageKey) as Ref<Language>;
+const systemLanguage = inject(systemLanguageKey) as Language;
```

Comment:
is this not also a ref?

Reply:
it's not -- we haven't implemented a language switcher in vue yet. and when we do, we'll want to stop using the arches object as the source of truth.